modin/pandas/test/dataframe/test_udf.py

```diff
@@ -305,6 +310,24 @@ def test_query(data, funcs):
         df_equals(modin_result, pandas_result)
 
 
+TEST_VAR = 2
```

Comment:
you have this defined at line 230, what's the point of re-defining?..

Reply:
removed